Barotrauma/BarotraumaShared/SharedSource/Items/Components/Machines/Sonar.cs

```diff
@@ -94,6 +82,27 @@ public float Range
             }
         }
 
+        [Serialize(2.0f, IsPropertySaveable.No, description: "The interval between active sonar pings.")]
+        public float PingInterval
+        {
+            get { return pingInterval; }
+            set { pingInterval = MathHelper.Max(value, 0.01f); }
+        }
+
+        [Serialize(5000f, IsPropertySaveable.No, description: "The speed at which sonar pings travel.")]
+        public float PingSpeed
+        {
+            get { return pingSpeed; }
+            set { pingSpeed = MathHelper.Max(value, 0.01f); }
+        }
```

Comment:
Imo it doesn't make sense for this to be editable, the pings should travel at a fixed speed (realistically should be the speed of sound, even though it's actually much slower in-game).